openshift/e2e-common.sh

```diff
@@ -100,18 +100,30 @@ function run_e2e_rekt_tests(){
   echo "Replacing images used in Rekt test resources with the images built in CI"
 
   echo "Replacing knative-eventing-test-event-library image"
-  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-library|${KNATIVE_EVENTING_TEST_EVENT_LIBRARY}|g" "$(dirname "$0")/../test/rekt/resources/eventlibrary/eventlibrary.yaml"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-library|quay.io/openshift-knative/eventing/event-library:v1.6|g" "$(dirname "$0")/../test/rekt/resources/eventlibrary/eventlibrary.yaml"
 
   echo "Replacing knative-eventing-test-heartbeats image"
-  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-heartbeats|${KNATIVE_EVENTING_TEST_HEARTBEATS}|g" "$(dirname "$0")/../test/rekt/resources/containersource/containersource.yaml"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-heartbeats|quay.io/openshift-knative/eventing/heartbeats:v1.6|g" "$(dirname "$0")/../test/rekt/resources/containersource/containersource.yaml"
 
   echo "Replacing knative-eventing-test-event-flaker image"
-  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-flaker|${KNATIVE_EVENTING_TEST_EVENT_FLAKER}|g" "$(dirname "$0")/../test/rekt/resources/flaker/flaker.yaml"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-event-flaker|quay.io/openshift-knative/eventing/event-flaker:v1.6|g" "$(dirname "$0")/../test/rekt/resources/flaker/flaker.yaml"
 
   echo "Replacing knative-eventing-test-eventshub image"
-  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-eventshub|${KNATIVE_EVENTING_TEST_EVENTSHUB}|g" "$(dirname "$0")/../vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml"
+  sed -i -e "s|registry.ci.openshift.org/openshift/knative-.*:knative-eventing-test-eventshub|quay.io/openshift-knative/eventing/eventshub:v1.6|g" "$(dirname "$0")/../vendor/knative.dev/reconciler-test/pkg/eventshub/103-pod.yaml"
 
-  go_test_e2e -timeout=1h -parallel=20 ./test/rekt || failed=$?
+  local test_name="${1:-}"
+  local run_command=""
+  local failed=0
+ 
+  if [ -n "$test_name" ]; then
+      local run_command="-run ^(${test_name})$"
+  fi
+  # check for test flags
+  local run_flags="-timeout=90m -parallel=20"
+  if [ -v EVENTING_TEST_FLAGS ] && [ ! -z "$EVENTING_TEST_FLAGS" ]; then
```

Comment:
I think this complicated condition could be converted to
` if [ -n EVENTING_TEST_FLAGS ]; then`